shared/src/main/scala/Elements.scala

```diff
@@ -477,6 +477,20 @@ object Elements:
         ListHelpers.map(a, ListHelpers.makeIterable(b, Some(true)))
     }
 
+    val merge: Dyad = addElem(
+      Dyad,
+      "J",
+      "Merge",
+      List("merge"),
+      "a: lst, b: lst -> Merge a and b",
+    ) {
+      case (a: VNum, b: VNum)   => MiscHelpers.eval(a.toString + b.toString)
+      case (a: VVal, b: VVal)   => MiscHelpers.add(a, b)
+      case (a: VVal, b: VList)  => VList.from(a +: b)
+      case (a: VList, b: VVal)  => VList.from(a :+ b)
+      case (a: VList, b: VList) => VList.from(a ++ b)
```

Comment:
```suggestion
      case (a: VList, b: VList) => VList.from(a ++ b)
      case (a, b: VList)  => VList.from(a +: b)
      case (a: VList, b)  => VList.from(a :+ b)
```
So that this works for functions too